static/app/views/issueList/overview.tsx

```diff
@@ -1085,13 +1089,13 @@ class IssueListOverview extends Component<Props, State> {
     actionType: 'Reviewed' | 'Resolved' | 'Ignored'
   ) => {
     if (itemIds.length > 1) {
-      addMessage(t(`${actionType} ${itemIds.length} Issues`), 'success', {
+      addMessage(`${actionType} ${itemIds.length} ${t('Issues')}`, 'success', {
```

Comment:
Kinda gross, but there's not really enough context here

Reply:
agree..but maybe we can refactor it in a follow up

Reply:
👍